content/post/stash-v2023.08.18/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Introducing Stash v2023.08.18
+date: "2023-08-22"
+weight: 10
+authors:
+- Hossain Mahmud
+tags:
+- backup
+- kubernetes
+- mongodb
+- mysql
+- restore
+- stash
+---
+
+We are announcing Stash `v2023.08.18` which includes a few bug fixes and Improvments. You can find the complete changelog [here](https://github.com/stashed/CHANGELOG/blob/master/releases/v2023.08.18/README.md). In this post, we are going to highlight the changes.
+
+### Bug Fixes
+- Resolved a bug causing MySQL `5.x.x` restoration failure in group replication mode. The issue occurred while restoring databases with *MyISAM* storage engine tables. To address this, we now exclude system databases from the backup process.
+([#198](https://github.com/stashed/mysql/pull/716))
+- Resolved a bug that was causing issues with shard backups in MongoDB versions `3.x.x`, `4.x.x`, and `6.x.x`. In the last release, some accidental changes were made. This new release fixes everything and makes sure things work correctly.
+
+### Improvements
+
+- We've now introduced backup and restore capabilities for MongoDB versions `5.0.2` and `5.0.3`.
```

Comment:
To resolve the `'_mergeAuthzCollections.tempRolesCollection' is missing but a required field` error while restoring,  We've now separated the backup-restore addon for MongoDB `5.0.x` early versions (namely 5.0.2 ,5.0.3) and later versions (5.0.15).  More details will be found [here](https://jira.mongodb.org/browse/TOOLS-2946).